docs: Revert "docs: haw-v25" (#2844)
OffchainLabs/arbitrum-docs: docs/how-arbitrum-works/04-state-transition-function/01-stf-gentle-intro.mdx

With <a data-quicklook-from="arbitrum">Arbitrum</a>, the STF plays an even more pivotal role. Offchain transactions are executed in batches according to this function, with the STF periodically submitting a concise summary of changes to the <a data-quicklook-from="parent-chain">parent chain</a>. This approach leverages offchain computation to boost throughput and reduce gas costs while anchoring the process to Ethereum's robust security model. To safeguard against incorrect or malicious offchain execution, Arbitrum employs a <a data-quicklook-from="challenge">challenge</a> mechanism known as fraud proofs. If a dispute arises, the STF can be recomputed step-by-step onchain, enabling the network to verify the validity of the offchain computations and ensure that the errors or fraudulent behavior are detected and rectified.

> Beyond these foundational aspects, the STF on the <a data-quicklook-from="arbitrum-nitro">Arbitrum Nitro</a> Stack closely mirrors Ethereum's STF in its overall structure but incorporates several key modifications to accommodate the unique requirements of the <a data-quicklook-from="arbitrum-chain">Arbitrum chain</a>. Essentially, the function receives a sequence of ordered transactions as its input, applies a defined set of rules to these inputs, and produces a new final state that reflects all the updates from the <a data-quicklook-from="batch">batch</a> of transactions.

Reviewer: ```suggestion
The STF on the <a data-quicklook-from="arbitrum-nitro">Arbitrum Nitro</a> stack also mirrors Ethereum's STF but incorporates several key modifications to accommodate <a data-quicklook-from="arbitrum-chain">Arbitrum chains'</a> requirements. Essentially, the function receives a sequence of ordered transactions as its input, applies a defined set of rules to these inputs, and produces a new final state that reflects all the updates from the <a data-quicklook-from="batch">batch</a> of transactions.
```

> With the introduction of <a data-quicklook-from="stylus">Stylus</a>, Arbitrum expands its execution model beyond the Ethereum Virtual Machine (EVM) by adding WebAssembly (<a data-quicklook-from="wasm">WASM</a>)–based smart contract execution. This enhancement allows high-performance smart contracts written in Rust, C, and C++ to run alongside traditional EVM contracts. The integration of Stylus introduces several modifications to the STF, including:

Reviewer: ```suggestion
With the introduction of <a data-quicklook-from="stylus">Stylus</a>, Arbitrum expands its execution model beyond the Ethereum Virtual Machine (EVM) by adding <a data-quicklook-from="wasm">WASM</a>–based smart contract execution. This enhancement allows high-performance smart contracts written in Rust, C, and C++ to run alongside traditional EVM contracts. Stylus introduces several modifications to the STF, including:
```
